app/code/Vendor/Voucher/view/frontend/templates/insert.phtml

```diff
@@ -0,0 +1,33 @@
+<style>
+    label {
+        display: inline-block;
+    }
+
+    input[type=text] {
+        display: block;
+        width: 50%;
+
+        border: 1px solid #dddddd;
+        background-color: #dddddd;
+        padding: 8px;
+    }
+</style>
+
+<h1>Add voucher status</h1>
+
+<?php $record = $block->getRecord(); ?>
+
+<form action="<?php echo $block->getFormAction(); ?>" method="post">
```

Comment:
You should use CSRF protection also.
https://magento.stackexchange.com/questions/131189/magento-2-what-is-the-use-of-formkey